doc/source/bindings.rst

```diff
@@ -291,6 +291,23 @@ Controls that must be used with LOCK
 Time
 ----
 
+MISSIONTIME, or MET
```

Comment:
Feel free to change it.  I mention "MET" down below in the paragraph, so that should be enough so it doesn't have to be in the title as well.